chain/blockValidator.go

```diff
@@ -309,6 +309,11 @@ func SignerCheck(header *block.Header) error {
 		return fmt.Errorf("invalid block signer public key %x, should be %x", header.UnsignedHeader.SignerPk, publicKey)
 	}
 
+	_, err = crypto.DecodePoint(header.UnsignedHeader.SignerPk)
```

Comment:
Can we remote this check?PubKey length check looks useless here.

Reply:
This is actually checking that `header.UnsignedHeader.SignerPk` is a valid hex string